.circleci/config.yml

```diff
@@ -98,11 +98,12 @@ jobs:
           # NOTE: -Werror disabled in CI, as we currently have >4k warnings.
           cd build \
             && cmake .. -DCMAKE_BUILD_TYPE=Debug -DENABLE_COVERAGE=On -DCPACK_GENERATOR=DEB \
-                -DENABLE_COMPILER_WARNINGS=On -DENABLE_WERROR=Off -DCMAKE_EXPORT_COMPILE_COMMANDS=On
+                -DENABLE_COMPILER_WARNINGS=On -DENABLE_WERROR=Off -DCMAKE_EXPORT_COMPILE_COMMANDS=On \
+                -DENABLE_ADDRESS_SANITIZER=ON -DENABLE_PYTHON_BINDINGS=Off
       - run: make -C build -j4
       - run: make -C build utrecht_tiles
       - run: make -C build -j2 tests
-      - run: make -C build -j2 check
+      - run: export ASAN_OPTIONS=detect_leaks=0 && make -C build -j2 check
```

Comment:
Can we have a short comment here on why the leak sanitizer is disabled? I know you have a comment in the README, but sometimes one might forget to look there, so might be easier to readers of the code to see an inline comment. Nothing elaborate, just a short one saying the leak sanitizer is disabled since its not supported on all platforms.

BTW, does enabling leak sanitizer cause the python bindings to fail to buid?

Reply:
Enabling leak sanitizer leads to errors that look very similar to what @kevinkreiser described in  [#2296 (comment)](https://github.com/valhalla/valhalla/pull/2296#issuecomment-608207370).

Python bindings to fail to build because of ASan itself. ASan requires some specific library load order which causes difficulties in case of python bindings.